app/mailboxes/inbox_mailbox.rb

```diff
@@ -0,0 +1,64 @@
+# typed: false
+
+class InboxMailbox < ApplicationMailbox
+  before_processing :required_info
+
+  def process
+    c = Comment.new(user: sending_user, comment: mail.decoded, is_from_email: true)
+    if parent.is_a?(Comment)
+      c.story_id = parent.story_id
+      c.parent_comment_id = parent.id
+    else
+      c.story_id = parent.id
+    end
+
+    # TODO: Figure out what to do here.
```

Comment:
What's this `TODO` about? Maybe this wants to just call `c.save!` so we get an exception in prod if a mail fails to parse?

It would be really nice if a failed mail could hang around in queue for a later bugfix/retry, but I'm hoping that a feature Action Mailbox has; it's probably not worth custom code.